Week1/exercise/w1/index.js

```diff
@@ -28,3 +44,13 @@ console.log('header: ', header);
 //    create <li> element
 //    set <li> element innerHtml to todoInput value
 //    add <li> element to todoList
+
+const addTodoButton = document.getElementById('btn-addTodo');
+const addTodo = () => {
+  const getTodoList = document.querySelector('#todoList');
```

Comment:
better to define these variables outside this function such that they are only selected once and not every time this function is called